rocAL/CMakeLists.txt

```diff
@@ -231,13 +232,16 @@ if(${BUILD_ROCAL})
                 include/augmentations/color_augmentations/
                 include/augmentations/effects_augmentations/
                 include/augmentations/geometry_augmentations/
+                include/augmentations/audio_augmentations/
```

Comment:
instead of adding each and every subdir name, we can fetch it as GLOB. We do this for other projects

Reply:
@rrawther @swetha097 I will make this change when I reorg the files for src/ directory and clean up include directory

Reply:
@LakshmiKumar23 I have made changes to use GLOB in CMake please check this [commit](https://github.com/ROCm/rocAL/pull/118/commits/47263d94393e9614a5cebcab6c11c804ac7152f8)